### DISCUSSION TITLE
tab id / module id

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1639] [Ideas]
for convenience sake, I would love to see the tabid and moduleid displayed somewhere within the UI.

**Comment by meetmandeep** [comment]
This will likely not happen. One of our goals for Vanjaro to keep the UI clean and intuitive. Tab ID and Module ID are useful for developers but meaningless for end users. It's easy enough to find the module id or tab id by looking at the page source (Inspect Element)
